hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestDirectoryDeletingServiceWithFSO.java

```diff
@@ -231,13 +231,14 @@ public void testDeleteWithLargeSubPathsThanBatchSize() throws Exception {
 
     assertSubPathsCount(dirDeletingService::getMovedFilesCount, 15);
     // 15 subDir + 3 parentDir
-    assertSubPathsCount(dirDeletingService::getMovedDirsCount, 18);
-    assertSubPathsCount(dirDeletingService::getDeletedDirsCount, 19);
+    // handled 5 move dir or file as capacity as recursive
+    assertSubPathsCount(dirDeletingService::getMovedDirsCount, 17);
+    assertSubPathsCount(dirDeletingService::getDeletedDirsCount, 25);
```

Comment:
Total number of subdirectories under appRoot i.e the directory on which delete is issued is 18 (3 immediate child * 5  + 3 + 1 (appRoot itself) makes the total recursive directory count 19.  Total directories = 19 but count is 25 ?

Reply:
Fixed, after refactoring, I was check for dirDeleted which is passed as null, but proto class is having as empty string, added that check to fix.